include/kpro_private.hrl

```diff
@@ -106,6 +106,12 @@
 -define(GET_STACKTRACE(Var), Var = erlang:get_stacktrace()).
 -endif.
 
+-ifndef(OLD_TIME_UNITS).
+-define(millisecond, millisecond).
+-else.
+-define(millisecond, milli_seconds).
```

Comment:
`milli_seconds` is not documented, use `millisecond` instead.

Reply:
It's been removed in OTP19, here's OTP18 documentation: http://erlang.org/documentation/doc-7.3//erts-7.3/doc/html/erlang.html#type-time_unit

P.S. note, it's if**n**def